examples/pln/ancestors/ancestors.scm

```diff
@@ -1,7 +1,7 @@
 ;; Load PLN (with only the rule that matters)
 (use-modules (opencog pln))
 (pln-load 'empty)
```

Comment:
BTW, this is no logger necessary (unless ones want to load the whole 'standard rule-base, which is probably a bad idea in general).

Reply:
Thank you, I though about it but was not confident.